akka-docs/src/main/paradox/java/scala-compat.md

```diff
@@ -1,37 +0,0 @@
-# Java 8 and Scala Compatibility
```

Comment:
Moving this to `paradox/scala/scala-compat.md `so that we consistently have .md files under the `paradox/scala` directory